evap/results/tests/test_views.py

```diff
@@ -52,6 +52,19 @@ def test_multiple_evaluations_per_course(self):
         self.assertNotContains(page, evaluation2.full_name)
         caches['results'].clear()
 
+
+    @patch('evap.evaluation.models.Evaluation.can_be_seen_by', new=(lambda self, user: True))
+    def test_order(self):
+        student = baker.make(UserProfile, email="student@institution.example.com")
+
+        course = baker.make(Course)
+        evaluation2 = baker.make(Evaluation, name_de='random_evaluation_b', name_en='random_evaluation_b', course=course, state='published')
+        evaluation1 = baker.make(Evaluation, name_de='random_evaluation_a', name_en='random_evaluation_a', course=course, state='published')
+
+        page = str(self.app.get(self.url, user=student))
```

Comment:
I'd slightly be in favor of using
```python
self.app.get(self.url, user=student).body.decode()
```
here -- sorry, I just saw the test I linked does it using `str()`. However, I feel that the string representation is more of a debug / user-facing thing, and we want just the html content, so this feels a bit cleaner to me.

(However, not a strong opinion, the current code also works)

Reply:
This is more elegant than using `str()`, changed it in the latest commit